packages/worker/src/api/controllers/global/users.ts

```diff
@@ -22,6 +22,65 @@ export const save = async (ctx: any) => {
   }
 }
 
+export const bulkSave = async (ctx: any) => {
+  let { users: newUsersRequested, groups } = ctx.request.body
+  let groupsToSave: any[] = []
+  const newUsers: any[] = []
+  const db = tenancy.getGlobalDB()
+  const currentUserEmails =
+    (await users.allUsers())?.map((x: any) => x.email) || []
+  for (const newUser of newUsersRequested) {
+    if (
+      newUsers.find((x: any) => x.email === newUser.email) ||
+      currentUserEmails.includes(newUser.email)
+    )
+      continue
```

Comment:
This continue is just floating, it should be wrapped with curlys for safety (hard to see immediately that the if statement only applies to the continue).